mycroft/skills/mycroft_skill/event_container.py

```diff
@@ -0,0 +1,157 @@
+from inspect import signature
+
+from mycroft.messagebus.message import Message
+from mycroft.metrics import Stopwatch, report_timing
+from mycroft.util.log import LOG
+
+from ..skill_data import to_alnum
+
+
+def unmunge_message(message, skill_id):
+    """Restore message keywords by removing the Letterified skill ID.
+    Arguments:
+        message (Message): Intent result message
+        skill_id (str): skill identifier
+    Returns:
+        Message without clear keywords
+    """
+    if isinstance(message, Message) and isinstance(message.data, dict):
```

Comment:
This isn't new code at all, this was safeties added for one reason or another way back. I believe it can be called with None as message in a couple of error cases... the message.data check is however redundant.